luxonis_ml/embeddings/methods/duplicate.py

```diff
@@ -1,79 +1,74 @@
-"""Near-duplicate Search with Qdrant.
+"""
+**Near-duplicate Search with Qdrant and Weaviate**
 
-This module provides utilities to detect and remove near-duplicate data points
-within a given set of embeddings. The removal process uses Kernel Density
-Estimation (KDE) on embeddings cosine similarity for optimal split, making
-this approach particularly suited for embeddings in high dimensional spaces.
+**Overview:**
```

Comment:
Use `B{Overview:}` for bold text instead of asterisks, see [epytext documentation](https://epydoc.sourceforge.net/epytext.html) for more details.